lib/structs/configuration-structs.go

```diff
@@ -32,13 +32,25 @@ type Kion struct {
 // Favorite holds information about user defined favorites used to quickly
 // access desired accounts.
 type Favorite struct {
-	Name                 string `yaml:"name"`
-	Account              string `yaml:"account"`
-	CAR                  string `yaml:"cloud_access_role"`
-	AccessType           string `yaml:"access_type"`
-	Region               string `yaml:"region"`
+	Name                 string `yaml:"name" json:"alias_name"`
+	Account              string `yaml:"account" json:"account_number"`
+	CAR                  string `yaml:"cloud_access_role" json:"cloud_access_role_name"`
+	AccessType           string `yaml:"access_type" json:"access_type"`
+	Region               string `yaml:"region" json:"account_region"`
 	Service              string `yaml:"service"`
 	FirefoxContainerName string `yaml:"firefox_container_name"`
+	CloudServiceProvider string `json:"cloud_service_provider"`
+}
+
+// FavoritesComparison holds the results of comparing local favorites with API
```

Comment:
Since this isn't a part of the config we should move this struct to the helper/transform.go file.